lab-jaren/server.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const http = require('http');
+const url = require('url');
+const querystring = require('querystring');
+const cowsay = require('cowsay');
+
+const parseBodyHandler = require('./lib/parse-body-handler.js');
+const badRequestHandler = require('./lib/bad-request-handler.js');
+const cowsayHTML = require('./lib/cowsay-html.js');
+
+const PORT = process.env.PORT || 3000;
+const server = http.createServer(function(message, response) {
+  /* parse url and query */
+  // url.parse turns "/api/cowsay?text=lulwat" ->
+  //    { pathname: "/api/cowsay", query: "text=lulwat" }
+  message.url = url.parse(message.url);
```

Comment:
I'm cool with you using `message` in place of `req` if that helps make more sense during the course of this class. Just know that ``req, res`` or ``request, response`` is usually convention, so just try to get comfortable with that common style while working with yours.